test/e2e/manifests/mysql/kustomization.yaml

```diff
@@ -3,12 +3,14 @@ kind: Kustomization
 
 resources:
   - ../../../../manifests/quick-start/mysql
+  - ../../../../manifests/quick-start/base/prometheus
```

Comment:
I don't think this will work, because RUN!=kubernetes typically , I think you only need to add to PROFILE=prometheus

Reply:
Sure thing